packages/orbit-components/src/Modal/ModalFooter/index.tsx

```diff
@@ -74,7 +74,7 @@ const ModalFooter = ({ dataTest, children, flex = "0 1 auto" }: Props) => {
         "duration-fast transition-shadow ease-in-out",
         "sm:max-lm:[&_.orbit-button-primitive]:h-form-box-normal sm:max-lm:[&_.orbit-button-primitive]:text-button-normal",
         childrenLength > 1 ? "lm:justify-between" : "lm:justify-end",
-        !isMobileFullPage && "lm:rounded-b-modal",
+        !isMobileFullPage ? "rounded-b-modal-mobile" : "lm:rounded-b-modal-mobile",
```

Comment:
Why is it changing from `rounded-modal` (9px) to `rounded-modal-mobile` (12px)? **EDIT:** It seems to be the value in Figma anyway, so this is not a question anymore, just the below one.

Also, since this is the footer, why does the `isMobileFullPage` affect the border-radius of the bottom border? It is always straight corners anyway, no? 🤔 

<img width="393" alt="image" src="https://github.com/kiwicom/orbit/assets/6265045/f9ce23a0-149c-4e9e-95ab-d97edea6528e">

Reply:
Yeah, I got confused.

I guess that just `lm:rounded-b-modal` would be enough? Regardless of `isMobileFullPage`?

Reply:
Seem so. But is it `lm:rounded-b-modal` or `lm:rounded-b-modal-mobile`?

Reply:
Let's wait for designers' answer. [This](https://github.com/kiwicom/orbit/pull/4258#discussion_r1487960071) was my assumption.

Reply:
After consulting with the Orbit designers, we confirmed that all the rounded corners must be 12px.